src/plrust.rs

```diff
@@ -21,6 +21,8 @@ use std::{
     process::Output,
 };
 
+use current_platform::CURRENT_PLATFORM;
+
```

Comment:
```suggestion
const CURRENT_PLATFORM: &str = env!("TARGET_PLATFORM");
```

Reply:
nit: Do you want TARGET_PLATFORM or HOST_PLATFORM?

There's no guarantee that HOST == TARGET always (even though it should be...)

Reply:
Eh, it doesn't matter 'cause the build.rs is annoying to also include, so it actually does makes sense to depend on another crate, thus why I merged it despite my initial grump.

And we do want the target platform, because that way we don't block a user fn from being compiled on one host and deployed on a different target.

Reply:
Ahh that's a good point. I was thinking about it mainly from the perspective of:

https://github.com/tcdi/plrust/blob/main/src/plrust.rs#L62

which you would want HOST_PLATFORM for?